src/core/astish.js

```diff
@@ -12,11 +12,11 @@ export let astish = (val) => {
 
     while ((block = newRule.exec(val.replace(ruleClean, '')))) {
         // Remove the current entry
-        if (block[4]) tree.shift();
-
-        if (block[3]) {
+        if (block[4]) {
+            tree.shift();
+        } else if (block[3]) {
```

Comment:
I imagine you have thought this through, but just in case the unit tests don’t cover this edge case: this code change modifies the logic when block[4] AND block[3] are truthy. Is that an oversight, or intentional?

Reply:
That's intentional; this can't happen. I'm not sure if it was possible in the past but a closing curly is in a separate RegEx alternation

Reply:
great :)

Reply:
🙌  hehe 😄  nice!